django_downloadview/response.py

```diff
@@ -117,9 +117,10 @@ class DownloadResponse(StreamingHttpResponse):
       attributes (size, name, ...).
 
     """
+
     def __init__(self, file_instance, attachment=True, basename=None,
                  status=200, content_type=None, file_mimetype=None,
-                 file_encoding=None):
+                 file_encoding=None, extra_headers=None, **kwargs):
```

Comment:
You're right, those are unrelated. `**kwargs` did come in from #109 and isn't necessary. And `extra_headers` came from some preliminary work on #122 that I didn't intend to include. I'll remove those unrelated changes.